packages/prisma-client/src/types.ts

```diff
@@ -1,9 +1,10 @@
-import type { Asset as DbAsset } from "@prisma/client";
+import type { Asset as DbAsset, Project as BaseProject } from "@prisma/client";
```

Comment:
It's a little bit confusing to me that things like `BaseProject` are pointing to different types in different files. It makes it harder to figure out the relationships between those things.

Reply:
half-migrated state, we have extracted prisma client into a separate package and now need to move all the augmented types to where they are are used, to the vertical/feature code